index.bs

```diff
@@ -1545,11 +1545,11 @@ for their editorial input.
             }
 
             return {
-                start: function() {
```

Comment:
These and the below are JS scripts that are not part of the spec. Changing them like this will break stuff.